components/infobox/commons/infobox_person.lua

```diff
@@ -66,6 +68,12 @@ function Person:createInfobox()
 		error('You need to specify an "id"')
 	end
 
+	if Logic.readBool(args.autoTeam) then
+		local team, team2 = PlayerIntroduction.playerTeamAuto()
```

Comment:
I'd personally prefer to pass in self.pagename here to have a single, defined source of that (it currently is always the same, both use `mw.title.getCurrentTitle().text`)
```suggestion
		local team, team2 = PlayerIntroduction.playerTeamAuto{player=self.pagename}
```
(As i said, personal preference, feel free to leave it as-is)